app/helpers/application_helper.rb

```diff
@@ -119,13 +119,19 @@ def holding_block_search args
     links = search_links(args[:document]['electronic_access_1display'])
     holdings_hash = JSON.parse(args[:document][args[:field]])
     holdings_hash.first(2).each do |id, holding|
-      render_arrow = !holding['library'].blank? and !holding['call_number'].blank?
+      render_arrow = (!holding['library'].blank? and !holding['call_number'].blank?)
       arrow = render_arrow ? ' &raquo; ' : ''
       info = ''
       if holding['library'] == 'Online'
-        info << content_tag(:span, 'ONLINE', class: 'availability-icon label label-primary',
-                            title: 'Electronic Access')
-        info << links.shift.html_safe
+        if links.empty?
+          info << content_tag(:span, 'LINK MISSING', class: 'label label-danger',
```

Comment:
@tampakis If we are rendering something with the value "Link Missing" should we be logging this somewhere so it can be corrected by cataloging. I'm assuming that "Link Missing is an error.